offchain/dusd-ui/views/components/footer.ejs

```diff
@@ -0,0 +1,84 @@
+<footer class="footer p-6">
+  <div class="columns is-vcentered is-mobile">
+    <div class="column">
+      <a href="/">
+        <img
+          src="../../assets/images/logo_footer.png"
+          alt="Ardana"
+          title="Ardana"
+        />
+      </a>
+    </div>
+
+    <div class="column is-narrow">
+      <div class="columns is-vcentered is-mobile">
+        <div class="column">
+          <span class="icon">
+            <a title="Twitter" class="has-text-white">
+              <svg role="none">
+                <use
+                  href="/assets/images/font-awesome-sprite-brands.svg#twitter"
+                />
+              </svg>
```

Comment:
```html
<a title="Twitter" class="has-text-white">
	<svg role="presentation" aria-hidden="true" focusable="false">
		<use href="/assets/images/font-awesome-sprite-brands.svg#twitter" />
	</svg>
	<span class="srOnly">Twitter</span>
</a>
```

or

```html
<a class="has-text-white">
	<svg role="presentation" focusable="false" aria-labelledby="title">
		<title>Twitter</title>
		<use href="/assets/images/font-awesome-sprite-brands.svg#twitter" />
	</svg>
</a>
```

Without either of these options, we effectively have `<img src=".." title="..">` and `<img>` without `alt` is problematic. If we want screen readers, SEO, TUIs, etc. to be able to read these icons, we need real text in the document. I like the second better, *but* the first works with more screen readers.

Reply:
great point, I've added title inside of `<svg>` tags